server/src/main/java/org/opensearch/index/codec/customcodecs/Lucene95CustomCodec.java

```diff
@@ -69,4 +75,8 @@ public StoredFieldsFormat storedFieldsFormat() {
     public String toString() {
         return getClass().getSimpleName();
     }
+
+    public void updateCompressionLevel(int compressionLevel) {
+        this.storedFieldsFormat = new Lucene95CustomStoredFieldsFormat(mode, compressionLevel);
```

Comment:
Can we use SetOnce semantics here to ensure once set, this should not be updated?

Reply:
I've updated!